userspace/otpilot/src/spi_processor.rs

```diff
@@ -149,7 +149,7 @@ impl<'a> SpiProcessor<'a> {
     fn process_spi_payload(&mut self, mut data: &[u8]) -> SpiProcessorResult<()> {
         let mut console = Console::new();
         let header = payload::Header::from_wire(&mut data)?;
-        writeln!(console, "Device: payload header: {:?}", header)?;
+        //writeln!(console, "Device: payload header: {:?}", header)?;
```

Comment:
Deleteme?

Reply:
Ah, that shouldn't have been deleted here. I'll undo it by overwriting this last commit.